scripts/yocto-build-and-test.sh

```diff
@@ -107,14 +107,8 @@ run_bitbake() {
 }
 
 prepare_and_set_PATH() {
-    # On branches without recipe specific sysroots, the next step will fail
-    # because the prepare_recipe_sysroot task doesn't exist. Use that failure
-    # to fall back to the old generic sysroot path.
-    if bitbake -c prepare_recipe_sysroot mender-test-dependencies; then
-        eval `bitbake -e mender-test-dependencies | grep '^export PATH='`:$PATH
-    else
-        eval `bitbake -e core-image-minimal | grep '^export PATH='`:$PATH
-    fi
+    bitbake -c prepare_recipe_sysroot mender-test-dependencies
```

Comment:
Fine cleanups!

Tbh, I have looked at these multiple times and been like: I should clean this up! :see_no_evil:

Reply:
I thought that I had already removed this a year ago...